src/pages/ServiceUnavailable/ServiceUnavailable.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import {Layout} from '../Layout/Layout';
+import MaintenanceInfoLayout from '../../components/MaintenanceInfo/MaintenanceInfoLayout';
+
+const ServiceUnavailable = () => (
+  <Layout>
+    <div>
+      <MaintenanceInfoLayout />
```

Comment:
This component does almost nothing, I would merge the code from `MaintenanceInfoLayout.tsx ` into this.